crawler/crawler.go

```diff
@@ -14,9 +19,24 @@ func Process(hosting Hosting, repositories chan Repository) {
 		return
 	}
 
-	err := hosting.ServiceInstance.GetRepositories(repositories)
+	sourceURL, err := hosting.ServiceInstance.GetRepositories(repositories)
 	if err != nil {
+		// Redis connection.
+		redisClient, err := redisClientFactory("redis:6379")
```

Comment:
Can we inject this as func argument? Also we should read the host (`redis:6379`) from env